SPECS/application-gateway-kubernetes-ingress/application-gateway-kubernetes-ingress.spec

```diff
@@ -0,0 +1,60 @@
+%global debug_package %{nil}
+Summary:        Application Gateway Ingress Controller
+Name:           application-gateway-kubernetes-ingress
+Version:        1.4.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Applications/Networking
+URL:            https://github.com/Azure/application-gateway-kubernetes-ingress
+#Source0:       https://github.com/Azure/application-gateway-kubernetes-ingress/archive/refs/tags/1.4.0.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/Azure/%%{name}/archive/refs/tags/%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2021-04-26 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#
+Source1:        %{name}-%{version}-vendor.tar.gz
+
+BuildRequires:  golang >= 1.13
+
+%description
+This is an ingress controller that can be run on Azure Kubernetes Service (AKS) to allow an Azure Application Gateway 
+to act as the ingress for an AKS cluster. 
+
+%prep
+%autosetup
+
+%build
+rm -rf vendor
+tar -xf %{SOURCE1} --no-same-owner
+
+export VERSION=%{version}
+export COMMIT=0e9dc17c
```

Comment:
Please add a comment explaining what this commit refers to. Should this be updated in the future versions? Same question for the `DATE` variable below.

If these values have to be updated with the version change, I'd suggest moving them to a `%define` at the top of the file with proper explanation of their purpose, so it's easier to find all the bits, which require a change and avoid any hard to catch bugs.

Reply:
Have removed the DATE variable as it's not mandatorily required to build. Have used %define for the git commit since that value will be updated if we upgrade the project version in the future.

Reply:
Thank you for the change. Please also say what this hash is for/why do we need it. Since it has to be updated for each version change, we should definitely mention that.

Btw. what happens if we build without the commit hash? I'm guessing you've already checked, since you were able to get rid of the commit date, but I want to make sure.

Reply:
It turns out that this git commit value is also not required for the binary to build. Only the version matters. Thanks for capturing this and I have removed the git hash tag in the latest commit.